cmake/modules/ddr/DDRSetStub.cmake.in

```diff
@@ -197,16 +211,31 @@ if(DDR_OVERRIDES_FILE)
 	list(APPEND EXTRA_DDRGEN_OPTIONS "--overrides" "${DDR_OVERRIDES_FILE}" WORKING_DIRECTORY "${override_dir}")
```

Comment:
I think `WORKING_DIRECTORY` should be specified independent of whether `DDR_OVERRIDES_FILE` is set, for example, the blacklist may be or other file names may be relative.

Reply:
THe issue I have with that is that it potentially changes behavior, when you specify the overrides file (the overrides / blacklist etc may reside in different directories). Instead, i have changed it so that relative file paths are all interred as relative to the directory where the ddr_set is declared

Reply:
I think it is reasonable that relative paths be interpreted based on the ddr_set directory. That suggest the name `override_dir` could be improved.